src/react/hooks/__tests__/useLazyQuery.test.tsx

```diff
@@ -1922,6 +1923,68 @@ describe("useLazyQuery Hook", () => {
       expect(options.fetchPolicy).toBe(defaultFetchPolicy);
     });
   });
+
+  test("regression for #11988: calling `clearStore` while a lazy query is running puts the hook into an error state and resolves the promise with an error result", async () => {
```

Comment:
```suggestion
  // regression for https://github.com/apollographql/apollo-client/issues/11988
  test("calling `clearStore` while a lazy query is running puts the hook into an error state and resolves the promise with an error result", async () => {
```

Perhaps we could just link to the issue in a comment? I tend to like keeping the test name clean 🙂

Reply:
I could swear the three tests above had the naming pattern, so I did it, but I must have imagined that :facepalm: